xvelmap/xr_accessor.py

```diff
@@ -63,6 +66,11 @@ def plot(self, u_var, v_var, lat_dim='latitude', lon_dim='longitude',
                     .format(var_name, (lat_dim, lon_dim), var_dims)
                 )
 
+            # Check if dataset contains nans. If so replace with 0
+            if np.any(xu.isnan(self._ds[var_name]).values):
```

Comment:
You could use numpy instead of `xarray.ufuncs`: ``np.any(np.isnan(self._ds[var_name].values))``.

But I don't think it is really need to first check for nans. ``DataArray.fillna()`` is not that expensive:

```python
In [1]: da = xr.DataArray(np.arange(100000000))

In [2}: %timeit da.fillna(0)
Out[2]: 815 ms ± 23.6 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Reply:
Agreed.